knimin/templates/pm_target_gene_library_prep.html

```diff
@@ -0,0 +1,281 @@
+{% extends logged_in_index.html %}
+{% block head %}
+{% from json import dumps %}
+<script type="text/javascript">
+
+  /**
+   *
+   * Submit the information
+   **/
+  function processDNAPlates() {
+    var plates = new Array();
+    var success = true;
+    $.each($('.entry'), function (idx, entry) {
+      var dnaSel = $(entry).find('.dna-plate:first');
```

Comment:
Good call - I think the Sel where there when I was struggling with the CSS to mark errors, but now they're no longer needed.